### PR TITLE
kdeconnect: update to 20.04.0

### PR DESCRIPTION
**Pull request by teldra**

### CODE REVIEW

**Comment by Chocimier** [comment]
In kde world, *.80 means pre-release. Stable 20.04.0 release is planned to next week, please update then.

**Comment by Chocimier** [comment]
Why is the patch needed?

**Comment by Johnnynator** [comment]
Pls use the distfile from  https://download.kde.org/unstable/release-service/20.03.90/src/kdeconnect-kde-20.03.90.tar.xz
 So smth like '${KDE_SITE/stable/unstable}/release-service/${version}/src/kdeconnect-kde-${version}.tar.xz', this allows xbps-src to use the KDE update check.

**Comment by teldra** [comment]
It is needed to allow kdeconnect to use this algorhythm. Afaik. Without, sshfs can not connect to my phone. Took the patch from archlinux. 
-- 
Diese Nachricht wurde von meinem Hubble-Teleskop gesendet.
Get my PGP key with: gpg --locate-keys oh@rotce.de

**Comment by teldra** [comment]
I was not aware oft this. Thx
-- 
Diese Nachricht wurde von meinem Hubble-Teleskop gesendet.
Get my PGP key with: gpg --locate-keys oh@rotce.de

**Comment by Chocimier** [comment]
Key exchange algorithms should be specified in configuration of ssh, not hardcoded in random executables.

**Comment by Johnnynator** [comment]
> Key exchange algorithms should be specified in configuration of ssh, not hardcoded in random executables.

kdeconnect forces ssh to not use any config. And the KDE Connect Android App only supports diffie-hellman-group14-sha1 and diffie-hellman-group1-sha1 [1].

[1] https://bugs.kde.org/show_bug.cgi?id=417787

**Comment by Chocimier** [comment]
Thanks for link! I still think we should be fixed upstream, not by us with patch.

**Comment by Johnnynator** [comment]
> Thanks for link! I still think we should be fixed upstream, not by us with patch.

Should be fixed in v1.14.1+ of the Android App.

**Comment by teldra** [comment]
> > Thanks for link! I still think we should be fixed upstream, not by us with patch.
> 
> Should be fixed in v1.14.1+ of the Android App.

I can confirm that!

**Comment by teldra** [comment]
Updated to 20.04.0. No patch needed.

File: common/shlibs [hunk 1]
```diff
@@ -3862,3 +3862,4 @@ libstk-4.6.1.so libstk-4.6.1_1
 libc++utilities.so.5 cpp-utilities-5.2.0_1
 libtagparser.so.9 tagparser-9.1.2_1
 libqtutilities.so.6 qtutilities-6.0.4_1
+libkdeconnectsmshelper.so.1 kdeconnect-20.04.0_1
```
Comment (Johnnynator) [inline common/shlibs:3865]: This entry isn't needed if not other packages links against it. Also common/shlibs should always end with a new line.

Comment (teldra) [reply common/shlibs]: Entry removed and re-added new line.

**Pull request merged**